broker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rabbitmq:3.13-management-alpine
+
+# No special configuration needed yet...
```

Comment:
We could explicitly set listening ports in rabbitmq.conf as a part of self documenting configuration